mne/decoding/transformer.py

```diff
@@ -245,6 +245,92 @@ def inverse_transform(self, X, y=None):
         return X.reshape(-1, self.n_channels, self.n_times)
 
 
+class Vectorizer(TransformerMixin):
+    """Transforms n-dimensional array into 2D array of n_sample by n_features.
+
+    MNE transformer have three, four dimensional output. This class converts
```

Comment:
Remove 'MNE transformer have ... output.", and rephrase the second second sentence into:

This class reshapes an n-dimensional array into an n_sample * n_features array, usable by the estimators and transformers of scikit-learn.